app/src/main/java/com/haruta/harutyan/originalapp/MainActivity.kt

```diff
@@ -3,14 +3,23 @@ package com.haruta.harutyan.originalapp
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
 import com.haruta.harutyan.originalapp.databinding.ActivityAddLocationBinding
 import com.haruta.harutyan.originalapp.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMainBinding
+    lateinit var db: AppDatabase
+
+    var locationList: List<Location> = emptyList()
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        db = AppDatabase.getInstance(this.applicationContext)!!
+        locationList = db.locationDao().getAll()
+        Log.d("room",locationList.size.toString())
```

Comment:
おそらく後ほど消すコードだと思うけど一応！
`emptyList` でリストを生成せずとも、 `onCreate` で変数を閉じ込めてあげれば良さそう！

```suggestion
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        db = AppDatabase.getInstance(this.applicationContext)!!
        val locationList = db.locationDao().getAll()
        Log.d("room",locationList.size.toString())
```